core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -1594,9 +1602,6 @@ object Chunk extends ChunkFactory with ChunkPlatformSpecific {
 
     val array: Array[A]
 
-    implicit val classTag: ClassTag[A] =
-      ClassTag(array.getClass.getComponentType)
```

Comment:
This can be a constant in most `Arr` implementations, which allows us to remove this `val` in most implementation (all except `AnyRefArray` actually)